examples/tv-casting-app/darwin/TvCasting/MatterBridge/CastingServerBridge.h

```diff
@@ -38,6 +42,15 @@
                                  clientQueue:(dispatch_queue_t _Nonnull)clientQueue
                        udcRequestSentHandler:(nullable void (^)(bool))udcRequestSentHandler;
 
+- (void)openBasicCommissioningWindow:(nullable void (^)(bool))commissioningCompleteCallback
```

Comment:
"commissioningComplete" in what sense?

Reply:
This is a _callback for when commissioning of this app has been completed (by usually an on-network
 TV/Media device acting as a Matter commissioner) via a call to the general commissioning cluster_. Added this documentation in the header. 

(Internally the CastingServer receives a DeviceEventCallback with the DeviceEventType::kCommissioningComplete.)